.github/workflows/docs.yml

```diff
@@ -26,6 +33,9 @@ jobs:
       - name: Build
         run: cd docs && npm install && hugo --minify
 
+      - name: ls
+        run: ls ./docs/public
+
       - name: Deploy 🚀
         uses: JamesIves/github-pages-deploy-action@a1ea191d508feb8485aceba848389d49f80ca2dc # v4.4.3
```

Comment:
if you want to add the pull_request, it is fine, but add a if condition here to be a push and from the main branch to not run this in a PR

Reply:
I don't think we want to do this on any pull_requests.